dev/com.ibm.ws.security.audit.utils/bnd.bnd

```diff
@@ -17,9 +17,15 @@ Bundle-Description: Security Audit: Security Audit Utils; version=${bVersion}
 WS-TraceGroup: audit
 
 Export-Package: \
-   com.ibm.ws.security.audit.*
+   com.ibm.ws.security.audit.utils
```

Comment:
Isn't this going to break things?  Things that were exported before will no longer be exported.

Reply:
the only package in this bundle is com.ibm.ws.security.audit.utils. So I modified it for clarification. I believe it won't break anything. Please let me know if you think it'll break anything. thanks.